### PR TITLE
Inital Report Generation

### PR DESCRIPTION
**Pull request by spencerpease**
This adds the functions used to generate a report from some template Rmarkdown file. Right now there is just one included template that doesn't take any parameters, but we can start working more useful templates.

### CODE REVIEW

**Comment by erinamay** [comment]
Makes sense to me. How many templates do you envision having?

**Comment by spencerpease** [comment]
Probably just 1 or 2, depending on the variety of diagnostics and plots we end up with.

**Review by erinamay — APPROVED** [review]
Makes sense to me. 

I reckon a report that just does level changes and then a second one that does both maybe?

(Sorry for adding a second review).

**Review by mobergm — APPROVED** [review]

File: inst/report_templates/test_report.Rmd [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
```
[the remark this answers is not in the document]
Comment (spencerpease) [reply inst/report_templates/test_report.Rmd:12]: Yep

**Pull request merged**